src/helpers/tests/geoChart/geoChart.spec.ts

```diff
@@ -66,13 +66,17 @@ describe("getGeoData", () => {
         );
     });
 
-    it("calculateAverage should return expected value ", () => {
-        const values: number[] = [1, 2, 3, 4, 5, 6];
-        expect(calculateAverage(values)).toEqual(3.5);
+    describe("calculateAverage", () => {
+        it("calculateAverage should return expected value ", () => {
```

Comment:
we should follow the convention: tests start with 'should ....'

Reply:
fixed